DQMOffline/Trigger/interface/HLTTauDQMOfflineSource.h

```diff
@@ -55,6 +58,10 @@ class HLTTauDQMOfflineSource : public DQMEDAnalyzer {
       edm::EDGetTokenT<LVColl> token;
     };
     std::vector<RefObject> refObjects_;
+    std::vector<edm::ParameterSet> tagAndProbePaths;
```

Comment:
hi @slehti - sorry for my slow answer - seems I missed your response

It looks like you can use
num_genTriggerEventFlag_.size()!=0

instead. if not, indeed, a simple bool is preferred.